src/soc/ibm/power9/Kconfig

```diff
@@ -0,0 +1,15 @@
+config CPU_IBM_POWER9
+	bool
+	help
+	  This example SoC code along with the example/min86 mainboard
+	  should serve as a minimal example how a buildable x86 SoC code
```

Comment:
It is not x86

Reply:
I have changed the help, to describe what this code is about.